config/runtime.exs

```diff
@@ -60,11 +60,14 @@ if sentry_dsn not in [nil, ""] do
     environment_name: env
 end
 
-scheduler_jobs =
-  if env == "prod",
-    do: [{"0 7 * * *", {Screenplay.Jobs.TakeoverToolTestingJob, :run, []}}],
-    else: []
-
-config :screenplay, Screenplay.Scheduler, jobs: scheduler_jobs
+if env == "prod" do
+  config :screenplay, Oban,
+    plugins: [
+      {Oban.Plugins.Cron,
+       crontab: [
+         {"* * * * *", Screenplay.Jobs.TakeoverToolTestingJob}
```

Comment:
I'm sure we could stand to run this more frequently, but running it every minute of every hour of every day seems excessive. Why the schedule change?

Reply:
Very much unintentional. Set it to that for the sake of testing and forgot to revert. Thank you.